packages/fstar/fstar.0.9.0/opam

```diff
@@ -0,0 +1,12 @@
+opam-version: "1"
+maintainer: "protz@microsoft.com"
+homepage: "http://fstar-lang.org"
+license: "Apache"
+depends: [
+  "ocamlfind"
+  "batteries"
```

Comment:
No specific version of batteries, although I suppose we could be defensive and limit to the specific version of batteries that we currently use. We just do `ocamlfind -package batteries` ... how can I find out which version of batteries this pulls?